File: RightslineDemoAppDotNet/RightslineDemoAppDotNetSQS/Config/ConfigSetup.cs
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Security.Cryptography;
using System.Threading.Tasks;
using System.Web;

namespace RightslineDemoAppDotNetSQS.Config
{
    public class ConfigSetup
    {  
        private const string Algorithm = "AWS4-HMAC-SHA256";
        public static Dictionary<string, string> GetConfigFile()
        {
            var config = new Dictionary<string, string>();
            var folderPath =
                AppContext.BaseDirectory.Substring(0,
                    AppContext.BaseDirectory.IndexOf("bin", StringComparison.Ordinal));
            var filePath = Path.Combine(folderPath, "Config\\config.json");
            try
            {
                var credentials = JObject.Parse(File.ReadAllText(filePath));
                var AccountId = credentials.GetValue("AccountId").ToString();
                var QueueName = credentials.GetValue("QueueName").ToString();
                var AccessKey = credentials.GetValue("AccessKey").ToString();
                var SecretKey = credentials.GetValue("SecretKey").ToString();
                var Region = credentials.GetValue("Region").ToString();
                
                config.Add("AccountId", AccountId);
                config.Add("QueueName", QueueName);
                config.Add("AccessKey", AccessKey);
                config.Add("SecretKey", SecretKey);
                config.Add("Region", Region);
                config.Add("Algorithm", Algorithm);
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
                Console.WriteLine("Please ensure that you have a valid config.json file in the " + folderPath +
                                  "Config folder.");
            }

            return config;
        }


    }
}

File: RightslineDemoApplication_Java/RightslineDemoAppJavaSQS/src/main/java/com/rightsline/ConfigSetup.java
package com.rightsline;

import com.google.gson.JsonObject;
import com.google.gson.JsonParser;


import java.io.File;
import java.io.FileReader;
import java.util.Base64;
import java.util.HashMap;

public class ConfigSetup {
    


    public static HashMap<String, String> getConfigFile(){
        HashMap<String, String> Credentials = new HashMap<>();
        File file = new File("./Config/dev_config.json");

        try {
            JsonObject credentials = new JsonParser().parse(new FileReader(file.getPath())).getAsJsonObject();
            String accessKey = credentials.get("AccessKey").getAsString();
            String secretKey = credentials.get("SecretKey").getAsString();
            String region = credentials.get("Region").getAsString();
            String queueName = credentials.get("QueueName").getAsString();
            String accountId = credentials.get("AccountId").getAsString();
            Credentials.put("AccountId", accountId);
            Credentials.put("QueueName", queueName);
            Credentials.put("AccessKey", accessKey);
            Credentials.put("SecretKey", secretKey);
            Credentials.put("Region", region);            
        }
        catch(Exception e){
            System.out.println("Please ensure that you have a valid config file in the Config folder ");
            e.printStackTrace();
        }
        return Credentials;


    }
}
